.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: node_js
 node_js:
   - '6'
+git:
+  depth: 5
```

Comment:
What's the difference between the default depth of `50` and using 5 instead?

Reply:
`5` is often faster

Reply:
Cloning 5 vs 50 commits makes a big difference in commit changes / diffs.

Reply:
How fast are we talking about compared to the whole build? Milliseconds or seconds?

Reply:
Just milliseconds so probably not worth it.